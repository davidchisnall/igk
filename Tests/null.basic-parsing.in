

\simpleCommand
\commandWithText{bar}
\commandOneArgument[a=b]{bar}
\commandOneQuotedArgument[a="b, c, d, \"fish\""]{bar}
\commandOneArgWithSpaces[a= b  c]{bar}
\commandTwoArguments[a= b  c , d=123]{bar}
text between
\commandTwoArgumentsNonASCII[a= 😁, d=12🇬🇧3]{bar\baz}
\commandEscapedBrace{ba\}r}
\commandEscapedSlash{ba\\r}

\nestedCommandOuter{text before\nestedCommandInner text after}
