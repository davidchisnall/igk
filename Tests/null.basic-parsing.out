

\simpleCommand{}
\commandWithText{bar}
\commandOneArgument[a=b]{bar}
\commandOneQuotedArgument[a="b, c, d, \\"fish\\""]{bar}
\commandOneArgWithSpaces[a=b  c]{bar}
\commandTwoArguments[d=123,a=b  c ]{bar}
text between
\commandTwoArgumentsNonASCII[d=12🇬🇧3,a=😁]{bar\baz{}}
\commandEscapedBrace{ba\}r}
\commandEscapedSlash{ba\\r}

\nestedCommandOuter{text before\nestedCommandInner{} text after}
